crates/hc_sandbox/src/calls.rs

```diff
@@ -484,8 +501,8 @@ async fn call_inner(client: &mut AdminWebsocket, call: AdminRequestCli) -> anyho
             // Print without other text so it can be piped
             println!(
                 "{}",
-                serde_json::to_string(
-                    &metrics
+                serde_json::to_value(
```

Comment:
I now see this is consistent with above but why convert to value and then to string?

Reply:
`serde_json::to_string()` internally converts to value first so since we need to use  `println!()` anyway which is going to convert to string , the only necessary part we need is `to_value()`. This makes a separation of concerns more explicit: if `to_value()` fails we know its exclusively related to the conversion to JSON and not to string.

Reply:
As detail for this, the method that's invoked to do the serializatin inside the library is this one https://docs.rs/serde_json/1.0.141/serde_json/ser/fn.to_writer.html

That explicitly documents that the input structure is serialized to the output stream.

Reply:
I'd prefer this wasn't a case of belief. The absolute most efficient way to do this is with `to_writer` and providing a handle to stdout but I think `to_string` is a reasonable middle ground of not making the code too clever and still being reasonably efficient. I think it's important that we can come to an agreement about things like that.